### PR TITLE
Favourite Fragment Implementation

### PR DESCRIPTION
**Pull request by pulkonet**

### CODE REVIEW

File: app/src/main/java/com/sdsmdg/pulkit/callingtext/TabsPagerAdapter.java [hunk 1]
```diff
@@ -33,12 +35,14 @@ public CharSequence getPageTitle(int position) {
...
                 return "New";
+            case 3:
+                return "Favourite";
         }
         return null;
     }
     @Override
     public int getCount() {
-        return 3;
+        return 4;
```
Comment (pkarira) [inline app/src/main/java/com/sdsmdg/pulkit/callingtext/TabsPagerAdapter.java]: Why 4? I think there are only 3 fragments

File: build.gradle [hunk 2]
```diff
@@ -24,6 +24,10 @@ allprojects {
         maven {
             url "https://jitpack.io"
         }
+        maven {
+            url 'http://dl.bintray.com/amulyakhare/maven'
```
Comment (pkarira) [inline build.gradle]: why is this added?

File: app/src/main/java/com/sdsmdg/pulkit/callingtext/TabsPagerAdapter.java (hunk 1 above)
Comment (pulkonet) [inline app/src/main/java/com/sdsmdg/pulkit/callingtext/TabsPagerAdapter.java]: I have made a new fragment for favourites. New fragment will collapse into dialer and there will only be three fragments at last.

File: build.gradle (hunk 2 above)
Comment (pulkonet) [inline build.gradle]: Library for icons.
